src/agent/onefuzz-agent/src/local/tui.rs

```diff
@@ -67,13 +67,6 @@ const TICK_RATE: Duration = Duration::from_millis(250);
 const FILE_MONITOR_POLLING_PERIOD: Duration = Duration::from_secs(5);
 const EVENT_POLLING_PERIOD: Duration = Duration::from_secs(1);
 
-#[derive(Debug, Default)]
-struct CoverageData {
```

Comment:
This is from WIP, and will very likely get used as-is. I suggest we suppress the lint for this with an [`allow` annotation](https://github.com/rust-lang/rust-clippy#allowingdenying-lints).